classes/task/opensesamesync.php

```diff
@@ -56,7 +56,10 @@ public function execute($testing = null): bool {
         mtrace("Opensesame task just started.");
 
         $opensesameapi = new opensesameapi;
-        $opensesameapi->get_auth_token();
+
+        if ($opensesameapi->authenticate()) {
```

Comment:
If authenticate call fails, how will we know from looking at logs?

Reply:
If the authenticate fails the scheduled task would not continue.  It would fail with a moodle exception. The same behavior as above would take place.